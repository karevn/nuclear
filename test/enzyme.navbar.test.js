import React from 'react';
import { shallow } from 'enzyme';

import Navbar from '../app/components/Navbar';

describe('<Navbar />', () => {
  it('renders with no children', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.children().length).toBe(0);
  });

  it('renders with children', () => {
    const wrapper = shallow(
      <Navbar>
        <div>First</div>
        <div>Second</div>
        <div>Third</div>
      </Navbar>
    );
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.children().length).toBe(3);
  });
});
