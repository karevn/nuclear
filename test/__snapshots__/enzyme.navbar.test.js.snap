// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> renders with children 1`] = `
<div
  className="navbar"
>
  <div>
    First
  </div>
  <div>
    Second
  </div>
  <div>
    Third
  </div>
</div>
`;

exports[`<Navbar /> renders with no children 1`] = `
<div
  className="navbar"
/>
`;
